ref/Arithmetic_Operators.md

```diff
@@ -318,10 +324,13 @@ The following operators generate pseudo random numbers.
 #### Uniformly distributed random real number between 0 and `‹number›`: `random(‹number›)`
 
     > random(10)
-    < 7.089078226464412
-    ~ \d\.\d{3,}
+    < 7.089
+    ~ \d\.\d+
+    > random(1000)
+    < 882.4784
+    ~ \d+\.\d+
     > sum(1..1000, random(5))
-    < 2464.8003929607607
+    < 2464.8004
     ~ 2\d\d\d\.\d+
```

Comment:
OK. I still prefer if test cases do not produce failures that are false negatives.

By the way, does travis CI retry a build if such a failure occurs? Or might this cause problems?